game-core/src/main/java/games/strategy/engine/data/MoveDescription.java

```diff
@@ -0,0 +1,77 @@
+package games.strategy.engine.data;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.HashMultiset;
+import com.google.common.collect.Maps;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import lombok.Getter;
+
+/** Describes an action that moves one or more units along a specific route. */
+public class MoveDescription extends AbstractMoveDescription {
+  private static final long serialVersionUID = 2199608152808948043L;
+  @Getter private final Route route;
```

Comment:
minor, if `@Getter` is on all variables, you can put it on the class and have it in one spot.